pkg/operator/rbac.go

```diff
@@ -13,8 +13,9 @@ import (
 )
 
 func AssertServiceAccount(clients *clients.Clients, ns, targetSA string) {
```

Comment:
IMHO we should unify the naming conventions for these `Assert...` functions... I like the one that you've just added (`AssertCronjobPresent` and `AssertCronjobNotPresent`). I'm OK doing it in a new PR.